ide/db/nbproject/project.properties

```diff
@@ -15,8 +15,20 @@
 # specific language governing permissions and limitations
 # under the License.
 
+auxiliary.org-netbeans-modules-editor-indent.CodeStyle.project.expand-tabs=true
+auxiliary.org-netbeans-modules-editor-indent.CodeStyle.project.indent-shift-width=2
+auxiliary.org-netbeans-modules-editor-indent.CodeStyle.project.spaces-per-tab=2
+auxiliary.org-netbeans-modules-editor-indent.CodeStyle.project.tab-size=4
+auxiliary.org-netbeans-modules-editor-indent.CodeStyle.project.text-limit-width=100
+auxiliary.org-netbeans-modules-editor-indent.CodeStyle.project.text-line-wrap=none
+auxiliary.org-netbeans-modules-editor-indent.CodeStyle.usedProfile=project
+auxiliary.org-netbeans-modules-editor-indent.text.x-java.CodeStyle.project.expand-tabs=true
+auxiliary.org-netbeans-modules-editor-indent.text.x-java.CodeStyle.project.indent-shift-width=4
+auxiliary.org-netbeans-modules-editor-indent.text.x-java.CodeStyle.project.spaces-per-tab=4
+auxiliary.org-netbeans-modules-editor-indent.text.x-java.CodeStyle.project.tab-size=4
+auxiliary.org-netbeans-modules-editor-indent.text.x-java.CodeStyle.project.text-limit-width=100
```

Comment:
probably not intended to commit module specific indentation settings?

Reply:
Yes, it's intentional, because it influences how Matisse form builder code is generated. The settings stored here ensure the code is generated with the same settings as earlier, simplifying diffs.

Reply:
Hmm, I see some settings that seem specific to my own IDE though. But removing any of them seemed to make them ineffective. I can just remove the file.

Reply:
interesting. I haven't considered global ide settings messing with codegen - but it makes sense of course.

we should still try to avoid adding those module specific settings since NB has [~800 projects](https://github.com/apache/netbeans/issues/7981#issuecomment-2501928011). Would be good to have a central place for this and let everything inherit it - but I don't think this is possible, otherwise we likely would have a central formatter file somewhere already. (I did see hint settings though)